Day_2_Software_engineering_best_practices/solutions/06_unit_tests/spectra_analysis.py

```diff
@@ -0,0 +1,289 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
you can remove this file (We should *maybe* also remove it in the modules solution, I started with this as one module, and only afterwards created the package with the same name.